server/schema/Resolvers.ts

```diff
@@ -1,16 +1,29 @@
+/* eslint-disable no-underscore-dangle */
 import { IResolvers } from '@graphql-tools/utils';
```

Comment:
Will review this file once #63 is closed in order to focus on auth specific changes.